R-package/man/RGF_Regressor.Rd

```diff
@@ -14,48 +13,9 @@
-
-\item{loss}{a character string specifying the \emph{Loss function}. One of \emph{"LS"} (Square loss), \emph{"Expo"} (Exponential loss) or \emph{"Log"} (Logistic loss).}
-
-\item{reg_depth}{a float. Must be no smaller than 1.0. Meant for being used with the algorithm \emph{RGF Opt} or \emph{RGF Sib}. A larger value penalizes deeper nodes more severely.}
-
-\item{l2}{a float. Used to control the degree of L2 regularization.}
-
-\item{sl2}{a float or NULL. Override L2 regularization parameter l2 for the process of growing the forest. That is, if specified, the weight correction process uses l2 and the forest growing process uses sl2. If NULL, no override takes place and l2 is used throughout training.}
-
-\item{normalize}{a boolean. If True, training targets are normalized so that the average becomes zero.}
-
-\item{min_samples_leaf}{an integer or a float. Minimum number of training data points in each leaf node. If an integer, then consider \emph{min_samples_leaf} as the minimum number. If a float, then \emph{min_samples_leaf} is a percentage and ceil(min_samples_leaf * n_samples) are the minimum number of samples for each node.}
-
-\item{n_iter}{an integer or NULL. The number of iterations of coordinate descent to optimize weights. If NULL, 10 is used for loss = "LS" and 5 for loss = "Expo" or "Log".}
-
-\item{n_tree_search}{an integer. The number of trees to be searched for the nodes to split. The most recently grown trees are searched first.}
-
-\item{opt_interval}{an integer. Weight optimization interval in terms of the number of leaf nodes. For example, by default, weight optimization is performed every time approximately 100 leaf nodes are newly added to the forest.}
-
-\item{learning_rate}{a float. Step size of Newton updates used in coordinate descent to optimize weights.}
-
-\item{memory_policy}{a character string. One of \emph{"conservative"} (it uses less memory at the expense of longer runtime. Try only when with default value it uses too much memory) or \emph{"generous"} (it runs faster using more memory by keeping the sorted orders of the features on memory for reuse). Memory using policy.}
-
-\item{verbose}{an integer. Controls the verbosity of the tree building process.}
-
-\item{init_model}{either NULL or a character string, optional (default=NULL). Filename of a previously saved model from which training should do warm-start. If model has been saved into multiple files, do not include numerical suffixes in the filename. \emph{NOTE:} Make sure you haven't forgotten to increase the value of the max_leaf parameter regarding to the specified warm-start model because warm-start model trees are counted in the overall number of trees.}
-
-\item{filename}{a character string specifying a valid path to a file where the fitted model should be saved}
-}
 \description{
+Regularized Greedy Forest regressor
```

Comment:
This line is needed?

Reply:
The previous .Rd file consisted of a [single line](https://github.com/RGF-team/rgf/blob/master/R-package/man/RGF_Regressor.Rd#L59). The .Rd file is the manual page of the corresponding .R file. It has to do with the [roxygen2](https://github.com/RGF-team/rgf/blob/master/R-package/DESCRIPTION#L24) package and how it builds the manual pages from the .R files (I've used the *roxygen2* 7.0.2 version for this PR)